### PR TITLE
Release test line results translation

### PR DESCRIPTION
**Pull request by andriichumak**

### CODE REVIEW

File: packages/smst-to-text/README.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+<pre>✔ Assert: Logo timeout 5s 
+✔ image            = ./../img/suitest-logo-mobile.svg
+✔ image load state = loaded                          </pre>
+<pre>✔ Send text test to element</pre>
+<pre>✔ Send text [[Enter]] to window until condition is met max 5x every 1s
+✔ Search results container is visible</pre>
+<pre>✖ Assert: First search result 
+✔ text       contains test            
+✖ text color =        rgb(200, 96, 96)
+       →        rgb(240, 96, 96)
```
Comment (JeKachan) [inline packages/smst-to-text/README.md:49]: small misalignments

File: packages/smst-to-text/README.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+<span style="color: green">✔ </span>image   = <span style="text-decoration: underline">./../img/suitest-logo-mobile.svg</span>
+<span style="color: green">✔ </span>image load state = <span style="text-decoration: underline">loaded</span>                  </pre>
+<pre><span style="color: green">✔ </span>Send text <span style="text-decoration: underline">test</span> to <span style="color: green">element</span></pre>
+<pre><span style="color: green">✔ </span>Send text <span style="text-decoration: underline">[[Enter]]</span> to <span style="color: green">window</span> until condition is met max <span style="text-decoration: underline">5</span>x every <span style="text-decoration: underline">1s</span>
+<span style="color: green">✔ </span><span style="color: green">Search results container</span> is visible</pre>
+<pre><span style="color: darkred">✖ </span>Assert: <span style="color: green">First search result</span> 
+<span style="color: green">✔ </span>text contains <span style="text-decoration: underline">test</span>    
+<span style="color: darkred">✖ </span>text color =        <span style="text-decoration: underline">rgb(200, 96, 96)</span>
+       →        rgb(240, 96, 96)
```
Comment (JeKachan) [inline packages/smst-to-text/README.md:69]: small misalignment

File: packages/smst-to-text/lib/toText.ts [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+/**
+ * Render a single text node as plain text
+ */
+const renderPlainTextNode: RenderTextFunc = (node?: ExtendedInlineNodes): string => node ? node.value : '';
+
+/**
+ * Render a single text node with ANSI styling
+ */
+const renderFormattedTextNode: RenderTextFunc = (node: ExtendedInlineNodes): string =>
+	formatString(node?.value ?? '', node?.type);
```
Comment (JeKachan) [inline packages/smst-to-text/lib/toText.ts:64]: for what needed checks for null and undefined? node declared as required argument.

Comment (andriichumak) [reply packages/smst-to-text/lib/toText.ts:64]: There is a misalignment in JSX factory, we'll need to update it in following release

**Pull request merged**